### PR TITLE
[GitHub] (Readme) Update readme with issue advice

### PR DESCRIPTION
**Pull request by AlexJerabek**
Clarifying the purpose of the repo and the types of issues to file here.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[README.md](https://github.com/OfficeDev/office-js-docs-pr/blob/ALexJ-Contribute/README.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812031944282661-595%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dFZX%2BNjs1uHGJ0%2B%2Fe97S7uVAmkPE7fZwPGecULbmzoQ%3D&st=2018-12-03T19%3A40%3A30Z&se=2019-01-03T19%3A45%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,31 @@
-# Office JavaScript APIs
+# Office Add-ins documentation
 
 Welcome to the Office JavaScript API conceptual documentation. In this repository, you can find the documentation source files for Office JavaScript API concepts, quick starts, tutorials, and how-to guides. For the best experience, we recommend you view this content on [docs.microsoft.com](https://docs.microsoft.com/office/dev/add-ins).
```
Comment (mscharlock) [inline README.md]: Suggestion: Given we are changing the title, maybe description should also mention add-ins? I know Office JS API conceptual docs and add-in docs are the same but if I came here without knowing that I might be confused?

Comment (kbrandl) [reply README.md]: Re Michelle's suggestion, you could simply change the first sentence to "Welcome to the Office Add-ins documentation repo."

**Comment by mscharlock** [comment]
Yep!

**Pull request merged**